src/utils/slugify.ts

```diff
@@ -0,0 +1,36 @@
+export function slugify(input: string): string {
```

Comment:
could you jsdoc the function, eg https://github.com/linz/argo-tasks/blob/master/src/utils/geotiff.ts#L6